src/Cake/Cake.csproj

```diff
@@ -42,6 +42,7 @@
     <RunCodeAnalysis>false</RunCodeAnalysis>
     <CodeAnalysisRuleSet>..\Cake.ruleset</CodeAnalysisRuleSet>
     <Prefer32Bit>false</Prefer32Bit>
+    <LangVersion>default</LangVersion>
```

Comment:
As there's a PR for  ` <LangVersion>5</LangVersion>` this change feels unnecessary, right?